windows/README.md

```diff
@@ -0,0 +1,77 @@
+# The Flutter Windows Desktop Embedder
+
+This framework provides the basic functionality for embedding a Flutter app on the Windows desktop.
+
+This README assumes you have already read [the top level README](../README.md), which contains important information and setup common to all platforms.
+
+## Flutter Location
+
+Contrary to [the top level README](../README.md) Windows cannot be run using a `.flutter_location_config` file and a Flutter tree must be located in the same parent directory as this repo like so:
+
+```
+<parent dir>
+  ├─ flutter (from http://github.com/flutter/flutter)
+  └─ flutter-desktop-embedding (from https://github.com/google/flutter-desktop-embedding)
+```
+
+## Adding Windows as a Target Platform
+
+By default flutter will throw an unknown target platform error if run on Windows. Windows can be added by editing `\path\to\flutter\packages\flutter\lib\src\foundation\platform.dart`.
+
+Change the code around line 37 from this:
+
+```dart
+if (Platform.isAndroid || Platform.isLinux) {
+  result = TargetPlatform.android;
+}
+```
+
+to this:
+
+```dart
+if (Platform.isAndroid || Platform.isLinux || Platform.isWindows) {
+  result = TargetPlatform.android;
+}
+```
+
+The flutter app you are trying to run must be rebuilt before it can be run on windows.
+
+```
+> flutter build bundle
+```
+
+## Using the Library
+
+Build the GLFW Library project in Visual Studio into a static or dynamic library, then link `flutter_embedder.lib` and include `embedder.h` into your binary. Also ensure that the `flutter_engine.dll`, and if using a dynamic library `flutter_embedder.dll`, are in valid DLL include paths.
+
+The output files are located in `bin\x64\$(Configuration)\GLFW Library\`.
+
+## Example Application
+
+The application under `GLFW Example\` shows an example application using the library.
+
+The example application must be built before running the windows executable as Visual Studio doesn't automatically do this.
```

Comment:
Since you're already running shell commands as pre-build steps, presumably this could be done fairly easily on Windows, as is currently done on the other platforms?

It makes the edit/run cycle much less annoying and error-prone when changing the Flutter application.